taxcalc/functions.py

```diff
@@ -184,11 +185,18 @@ def CapGains(p23250, p22250, _sep, ALD_Investment_ec, ALD_StudentLoan_HC,
     c23650 = p23250 + p22250
     # limitation on capital losses
     c01000 = max((-3000. / _sep), c23650)
-    # compute ymod* variables
+    # compute ymod1 variable that is included in AGI
     ymod1 = (e00200 + e00700 + e00800 + e00900 + e01400 + e01700 +
              (1 - ALD_Investment_ec) * (e00300 + e00600 +
                                         c01000 + e01100 + e01200) +
              e02000 + e02100 + e02300)
+    if CG_nodiff != 0.:
```

Comment:
Some users do want to have both preferential rates and an exclusion. For example, the current House Republican [Blueprint](https://abetterway.speaker.gov/_assets/pdf/ABetterWay-Tax-PolicyPaper.pdf) includes both preferential rates and a 50% exclusion. I believe @codykallen recently modeled this by just cutting the rates in half, but if we have a parameter called `qdcg_exclusion`, then some users might abuse it and try to implement it when CG_nodiff = 0. There is a warning in the notes about not doing that, but many users don't read the notes carefully and they won't receive any other warning that the reform they thought they were implementing wasn't implemented. 

If it adds significant complexity to relax the constraint that CG_nodiff!=0, then I understand taking the risk that some users make mistakes, but if it is simple to relax the constraint, then it could enhance usability.